lib/listener/helpers.js

```diff
@@ -11,9 +11,16 @@ module.exports = function () {
   const helpers = container.helpers();
 
   const runHelpersHook = (hook, param) => {
-    if (store.dryRun) return;
+    if (store.dryRun) 
+      return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected no linebreak before this statement.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4932579)